sdk/containerregistry/azure-containers-containerregistry/src/main/java/com/azure/containers/containerregistry/implementation/authentication/AccessTokenCacheImpl.java

```diff
@@ -160,6 +173,80 @@ private Supplier<Mono<? extends AccessToken>> retrieveToken(TokenRequestContext
         };
     }
 
+    private Supplier<AccessToken> retrieveTokenSync(TokenRequestContext tokenRequestContext,
```

Comment:
We should unskip the auth function from asserting client tests now https://github.com/Azure/azure-sdk-for-java/blob/main/sdk/containerregistry/azure-containers-containerregistry/src/test/java/com/azure/containers/containerregistry/ContainerRegistryClientIntegrationTests.java#L59

Reply:
great catch, thanks!